tools/server/webui/src/components/useChatExtraContext.tsx

```diff
@@ -38,7 +38,7 @@ export function useChatExtraContext(): ChatExtraContextApi {
       }
 
       if (mimeType.startsWith('image/') && mimeType !== 'image/svg+xml') {
```

Comment:
maybe we should convert SVG to PNG, probably we will need to use browser Canvas API

Reply:
Done in https://github.com/ggml-org/llama.cpp/pull/13393/commits/345890710cbf859f9f9d8010dc1bf1b36e8246f4